docs/learning/README.md

```diff
@@ -1,41 +1,69 @@
-# Learning Pocket
+# Learning Pocket <!-- omit in toc -->
 
 _This is a live document on how to get ramped up on all the knowledge you need to contribute to the Pocket Protocol._
 
+- [Where to Start?](#where-to-start)
+- [Technical Foundation](#technical-foundation)
+  - [Github Development](#github-development)
+  - [Golang](#golang)
+- [Technical References](#technical-references)
+  - [Pocket Specific](#pocket-specific)
+  - [Consensus](#consensus)
+  - [Merkle Trees](#merkle-trees)
+  - [Ethereum](#ethereum)
+  - [Cryptography](#cryptography)
+  - [P2P](#p2p)
+  - [Blogs](#blogs)
+    - [Blogs - Specific Articles](#blogs---specific-articles)
+    - [Blogs - General Subscription Suggestions](#blogs---general-subscription-suggestions)
+- [Non-suggested reads](#non-suggested-reads)
+
 ## Where to Start?
 
 This is a general set of steps we have found to help new core team members onboard to Protocol development.
 
-1.  Watch our [2022 Infracon presentation on v1](https://www.youtube.com/watch?v=NJoZyzQuJVc) to get a general idea of how everything works.
-2.  Run a LocalNet by following the instructions [here](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md).
+1. Watch our [2022 Infracon presentation on v1](https://www.youtube.com/watch?v=NJoZyzQuJVc) to get a general idea of how everything works.
+2. Run a **LocalNet** by following the [development guide](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md).
 
-    **Pro tip**: Did you know there are a bunch of `make` commands? Run `make` from the root repo to see what’s available.
+   _TIP: Run `make` from the root repo to see the different commands available._
```

Comment:
The “Pro tip” is interesting, but it’s better to specify
    * Once you cloned the GitHub repository of pocket, you can `$cd pocket`, and then run `$make` to see all available commands.